.prettierrc

```diff
@@ -12,6 +12,23 @@
         "parser": "markdown",
         "proseWrap": "always"
       }
+    },
+    {
+      "files": "templates/**/*.*",
```

Comment:
I want the template files to be formatted and using 80 as width looks good enough standard to me.

Reply:
Using width of 80 is quite outdated, common practice is to use 120 nowadays.

Reply:
We use 80 for solidity files even in Airnode. I don't think it makes a big difference though since prettier is not included in the template anyway.

So it's up to the user to pick whatever standard he wants.